sklearn/utils/_metadata_requests.py

```diff
@@ -663,19 +663,18 @@ def __str__(self):
 
 # A namedtuple storing a single method route. A collection of these namedtuples
 # is stored in a MetadataRouter.
-MethodPair = namedtuple("MethodPair", ["callee", "caller"])
+MethodPair = namedtuple("MethodPair", ["caller", "callee"])
 
 
 class MethodMapping:
-    """Stores the mapping between callee and caller methods for a router.
+    """Stores the mapping between caller and callee methods for a router.
 
     This class is primarily used in a ``get_metadata_routing()`` of a router
     object when defining the mapping between a sub-object (a sub-estimator or a
-    scorer) to the router's methods. It stores a collection of ``Route``
-    namedtuples.
+    scorer) to the router's methods. It stores a collection of namedtuples.
 
-    Iterating through an instance of this class will yield named
-    ``MethodPair(callee, caller)`` tuples.
+    Iterating through an instance of this class will yield several of the
+    following `namedtuple`: ``MethodPair(caller, callee)``.
```

Comment:
it only yields several tuples if there are that many. So the previous wording was more accurate.

Reply:
I agree.